### PR TITLE
Syndicate mob spawn event rework.

### PR DESCRIPTION
**Pull request by Zostroll**
Changes the minimum amount of people from 15 to 20.
Decreases the event weight from 2 to 1. 
Makes the syndicate energy sword mobs be able to be hit by bullets, a 50% chance of laser hits, and decreases there heath from 170 to 130.
Reduces the sound of the event from 100 to 75 to stop it from being so painful for headphone users.
Makes the gear from dead syndicate retrievable. 
Reasoning: https://github.com/yogstation13/yogstation/issues/441
:cl:
tweak: Portal storm event has been reworked and the energy syndicate sword mobs have been nerfed
/:cl:

### CODE REVIEW

**Comment by Notamaniac** [comment]
Reasoning?

**Comment by GrayRachnid** [comment]
Fix it instead of removing it.

**Comment by FluffySurvivor** [comment]
:-1:

**Comment by Stealthkibbler** [comment]
> The event has a chance of summoning narsie btw, so shut the fucker down, NOW

:+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1:

**Comment by Nirnael** [comment]
Disregard what Adam said, he has no idea. The "narsie" event doesn't spawn narsie, it spawns 20 constructs and it's already disabled.

File: code/modules/events/portal_storm.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/portal_storm_syndicate
 	name = "Portal Storm: Syndicate Shocktroops"
 	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
-	weight = 2
+	weight = 0 // Overpowered
```
Comment (Nirnael) [inline code/modules/events/portal_storm.dm]: 0 weight doesn't disable the event, it just makes it really fucking hard to happen. If you want to disable an event you set its max_occurences to 0.

**Comment by X-TheDark** [comment]
Indeed, you can improve this event (instead of removing it) by replacing the proc at https://github.com/yogstation13/yogstation/blob/master/code/modules/mob/living/simple_animal/hostile/syndicate.dm#L58
with

```
/mob/living/simple_animal/hostile/syndicate/melee/bullet_act(obj/item/projectile/Proj)
    if(!Proj)
        return
    switch(Proj.damage_type)
        if(BRUTE)
            adjustHealth(Proj.damage)
        if(STAMINA) //No disabler buff
            visible_message("<span class='danger'>[src] seems unaffected by [Proj]!</span>")
        else
            if(prob(50))
                adjustHealth(Proj.damage)
            else
                visible_message("<span class='danger'>[src] blocks [Proj] with its shield!</span>")
    return 0
```

This would make bullets (except TOXIN type bullets, but details, details, right?) never be blocked, lasers be blocked with 50% chance.

As of note, EMT is incorrect in his statement that thrown weapons are blocked. They aren't, only projectiles are blocked.

**Comment by ShadowDeath6** [comment]
as much as I love random events that kill a lot of people, this is almost the same tier as blob considering that the syndicates do stupidly high damage and there are so many and they're so spaced out that they'll mostly wipe out anyone without a ballistic weapon, which aren't exactly that common. The best fighting you're going to get is from the chaplain with a sharpened x2 holy weapon and security with their shitty riot shotguns, and then people will be able to loot the eswords and eshields and hardsuits off the dead bodies, which obviously creates the problem of possible traitors with free nukeops tier shit for free

**Comment by Zostroll** [comment]
I have taken your advice and changed from removing it to a reworking. This is my first time coding anything and I thought I might start by fixing a bug report that I thought only needed one number changed it has turn out to be more than that for a rework. I have no coding experience and have done these changes under your advice if I have done some thing wrong in the rework please inform me.

**Comment by FluffySurvivor** [comment]
This is not a rework, this is a hardcore nerf.

**Comment by Zostroll** [comment]
Yeah its a nerf... I played a round were 80% of the crew died from this event. That shouldn't happen on a event that offers no ghost roles!

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm [hunk 2]
```diff
@@ -52,18 +52,23 @@
...
+        if(BRUTE)
+            adjustHealth(Proj.damage)
+        if(STAMINA) //No disabler buff
+            visible_message("<span class='danger'>[src] seems unaffected by [Proj]!</span>")
+        else
+            if(prob(50))
+                adjustHealth(Proj.damage)
+            else
+                visible_message("<span class='danger'>[src] blocks [Proj] with its shield!</span>")
+    return 0
```
Comment (Nirnael) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: tabulations reeeeeeeee

**Comment by Super3222** [comment]
> Reduces the sound of the event from 100 to 75 to stop it from being so painful for headphone users.

Lame. They should shit themselves. The Syndicates are coming!

**Comment by Time-Green** [comment]
We need to remove this completely, it just doesn't fit with yogs.

**Comment by Zostroll** [comment]
Tim I was originally removing it but a rework was advocated for if others just want it removed instead of nerfed I can do that.

**Comment by ShadowDeath6** [comment]
It really doesn't belong, and no amount of tinkering is worth fixing "die or get free nukeops shit", because I doubt it'll be much more than that.

**Comment by FluffySurvivor** [comment]
Fighting the syndie mobs is fun af

**Comment by GrayRachnid** [comment]
> It really doesn't belong, and no amount of tinkering is worth fixing "die or get free nukeops shit", because I doubt it'll be much more than that.

Then people should learn to adapt to it. It took me one fight with this random event to know everything about it's mechanics.

**Comment by X-TheDark** [comment]
By the way, don't worry, even if you don't use the `/noloot` versions of these mobs, they still don't spawn any loot, they'll just gib gruesomely on death, which is not too bad.

So these won't drop shields/swords for the crew to ~~powergame with~~ enjoy. Only bosses drop gear still.

**Comment by ghost** [comment]
If you keep it, I hope you enjoy having no fun at all @Fluffysurvivor

Keeping the syndie gear or weapons on you will net you a ban under the do not powergame rule if you're normal crew and your life is not actively in danger or under syndie gear into armory for regular security.

So you don't get to keep any of your cool loot. You loot the mobs and have to deconstruct it or space it.

**Comment by sagattyy** [comment]
replace the syndicate with creatures(morph, headslug and that creature from xenobio/wizard den)
done, no need to nerf it

**Comment by ParagonPrime** [comment]
I was on the fence about this before, but since [THIS](https://forums.yogstation.net/index.php?threads/the-syndicate-revolution.11275) just happened I'm definitely going to have to say that this event needs a huge fucking nerf.

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm (hunk 2 above)
Comment (X-TheDark) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: Yeah, you might want to replace all these spaces with TABs, so it's consistent with every other piece of code.

**Comment by oisin100** [comment]
Changelog

**Comment by Zostroll** [comment]
Changeloged

**Comment by oisin100** [comment]
Merge conflicts, And the changelog is incorrect.
the last CL in the changelog should be
`\:cl:`

**Comment by Zostroll** [comment]
How do I see whats conflicting?

**Comment by ShadowDeath6** [comment]
use the terminal and type
`git merge yogstation/master`, it'll tell you what conflicts

also in the cl you want to put tweak or something before the entry, so something like
tweak: Portal storm event has been reworked and the energy syndicate sword mobs have been nerfed

**Comment by Zostroll** [comment]
Umm what terminal?

**Comment by X-TheDark** [comment]
@Silk-Touch 
Merge conflict is caused by https://github.com/yogstation13/yogstation/commit/0dc36a7cb4cc924a51c6bc10caf529919a2b84b6
To be more precise, check the `syndicate.dm` changes in that commit I linked. Cruix changed the flags from **bitflags** (that your PR still uses) to a **list**. You need to change the `status_flag` variables in this PRs `syndicate.dm` to fix the merge conflict.

**Comment by Zostroll** [comment]
@X-TheDark I did that it is still conflicting.

**Pull request closed without merging**